askup/models.py

```diff
@@ -38,6 +69,24 @@ class Qset(models.Model):
     users = models.ManyToManyField(User, blank=True)
     questions_count = models.PositiveIntegerField(default=0, db_index=True)
 
+    class Meta:
+        unique_together = ('parent_qset', 'name')
+        verbose_name = 'Subject'
+        verbose_name_plural = 'Subjects'
+
+    @property
+    def full_name(self):
+        """
+        Return a string representation of a Qset object.
+
+        Tries to get the include_parent_name value first (can be passed by RawSQL)
+        in lack of it gets name field of model.
+        """
+        if getattr(self, 'include_parent_name', None):
```

Comment:
it seems this if statement should be rewritten.

Reply:
redone